runner.sh

```diff
@@ -276,7 +277,7 @@ for file in "${files[@]}"; do
 		done
 
 		# echo "$notebook" > "${base_path}/debug_notebook.ipynb"
-		time out=$(echo "$notebook" | papermill --progress-bar)
+		time out=$(echo "$notebook" | papermill --progress-bar -k python3)
```

Comment:
Do you think the kernel will always be called `python3`? Seems like something we should have as input argument to `runner.sh`. That said, I just ran `jupyter kernelspec list` locally and it's called `python3`, too. So maybe this is safe.

Reply:
For Python 3 I think so. I've added the option to the script anyway.